definitions/scenarios/upgrade_to_satellite_6_10.rb

```diff
@@ -64,16 +64,12 @@ def check_var_lib_pulp
     end
 
     def pulp3_switchover_steps
-      add_step(Procedures::Service::Start)
       add_step(Procedures::Service::Enable.
           new(:only => Features::Pulpcore.pulpcore_migration_services))
-      add_step(Procedures::Service::Start.
-          new(:only => Features::Pulpcore.pulpcore_migration_services))
-      add_step(Procedures::Content::Switchover.new(:skip_deb => true))
+      add_step(Procedures::Service::Start)
+      add_step(Procedures::Content::Switchover.new)
```

Comment:
Why did `:skip_deb => true` go away?

Reply:
the installer isn't actually getting run here downstream as part of switchover, so the parameter isn't needed